wakeonlan.py

```diff
@@ -56,7 +58,13 @@ def send_magic_packet(
     """
     packets = [create_magic_packet(mac) for mac in macs]
 
-    with socket.socket(socket.AF_INET, socket.SOCK_DGRAM) as sock:
+    try:
+        if isinstance(ipaddress.ip_address(ip_address), ipaddress.IPv6Address):
+            address_family = socket.AF_INET6
+    except ValueError:
+        pass
```

Comment:
Could you use `contextlib.ignored()` instead of a try/catch?

Reply:
I think you meant `contextlib.suppressed()` here, but I had to change this bit so I don't think you'd still want that in the new revision. No overstepping the user supplied value if the caller chooses to provide an explicit address_family now.

Reply:
Yep. That’s what I meant, but I think your new changes are even better.